### PR TITLE
fixed yaml syntax

### PR DESCRIPTION
**Pull request by helios-ag**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,22 +5,19 @@ php:
   - 5.5
   - 5.6
   - hhvm
-
+  - 7.0
```
Comment (alquerci) [inline .travis.yml]: @helios-ag are you sure there is a PHP 7.0?

File: .travis.yml [hunk 2]
```diff
@@ -5,22 +5,19 @@ php:
...
 
 matrix:
   allow_failures:
     - php: 5.6
     - hhvm
-
+    - 7.0
```
Comment (alquerci) [inline .travis.yml]: Good syntax here should be:
```yaml
    - php: 5.6
    - php: hhvm
    - php: 7.0
```
Or maybe this one to remove duplication:
```yaml
    - php:
      - 5.6
      - hhvm
      - 7.0
```

File: .travis.yml (hunk 1 above)
Comment (helios-ag) [inline .travis.yml]: night build already available :)

File: .travis.yml (hunk 2 above)
Comment (helios-ag) [inline .travis.yml]: thank you!

Comment (alquerci) [inline .travis.yml]: The second solution does not work sorry.

File: .travis.yml (hunk 1 above)
Comment (alquerci) [inline .travis.yml]: Ok so I need to learn about that.

**Pull request merged**